tests/ReflectionClassTest.php

```diff
@@ -290,4 +300,58 @@ protected function getGettersToCheck()
 
         return $allNameGetters;
     }
+
+    /**
+     * Returns list of ReflectionProperty that be checked
+     *
+     * @return array
+     */
+    protected function getPropertiesToCheck()
+    {
+        $names = [
+            'name',
+        ];
+
+        return $names;
+    }
+
+    /**
+     * @param string[] $membersToCheck
+     * @return array
+     */
+    protected function caseProvider($membersToCheck)
+    {
+        $testCases = [];
+        $files = $this->getFilesToAnalyze();
+        foreach ($files as $fileList) {
+            foreach ($fileList as $fileName) {
+                $fileName = stream_resolve_include_path($fileName);
+                $fileNode = ReflectionEngine::parseFile($fileName);
+
+                $reflectionFile = new ReflectionFile($fileName, $fileNode);
+                include_once $fileName;
+                foreach ($reflectionFile->getFileNamespaces() as $fileNamespace) {
+                    foreach ($fileNamespace->getClasses() as $refClass) {
+                        $qcn = ltrim($refClass->getName(), '\\'); // workaround for #80
+                        $fqcn = '\\' . $qcn;
+                        $classes = [
+                            $qcn => [$refClass, new \ReflectionClass($qcn)],
+                            $fqcn => [new ReflectionClass($fqcn), new \ReflectionClass($fqcn)],
+                        ];
+                        foreach ($classes as $caseName => list($parsedClass, $originalClass)) {
```

Comment:
1. Is this `list(` function usage compatible with all PHP versions where library itself can be used?
2. Please rename `$originalClass` into `$builtinClass`. Maybe it makes sense to rename corresponding parameter in method, where this data provider is used (e.g. `$refMethod` to `$builtinMethod` and such).

Reply:
1. I used list() in PHP3 era :) Changes in 7.0 does not affect this simple use case.
2. ok

Reply:
1 - you could use `list(`, but support for using `list` right within `foreach` construct was added later

Reply:
I just checked this, and it appears to work in [PHP 5.5.0 and above](https://3v4l.org/37Ok9). (I don’t think we support anything older than 5.6.0, so I suspect this will be ok. @aik099, can you confirm this?)

Reply:
What I've meant is for @VolCh to find http://php.net/manual/en/control-structures.foreach.php#control-structures.foreach.list, which clearly says that it works since PHP 5.5. That doesn't quite qualify for `I used list() in PHP3 era` :)

Yeah, @loren-osborn , it works according to docs. I haven't used such syntax myself though.